index/scorch/scorch.go

```diff
@@ -223,7 +223,7 @@ func (s *Scorch) openBolt() error {
 	s.introducerNotifier = make(chan *epochWatcher, 1)
 	s.revertToSnapshots = make(chan *snapshotReversion)
 	s.persisterNotifier = make(chan *epochWatcher, 1)
-
+	s.closeCh = make(chan struct{})
```

Comment:
minor thing... I did happen to like the previous empty whitespace line between initializing all the s.this and s.that "paragraph" of code

Reply:
done!